event-based-simulators/extras/GenerateMeasurements.py

```diff
@@ -0,0 +1,248 @@
+import sys
+import json, os, logging, requests, base64
+from datetime import datetime, timedelta, time
+from random import randint, uniform, choices, gauss
+from statistics import mean
+
+from c8y_api import CumulocityApi
+
+import ArgumentsAndCredentialsHandler, Environment
+
+C8Y_PROFILE_GROUP = 'c8y_EventBasedSimulatorProfile'
```

Comment:
Please go with a standard way of string quotations. Chose either and be consistent in the script